### ISSUE TITLE
0.3.3 (707d415) Fountains in game do not animate

### ISSUE THREAD

**Issue by darth-crunchus** [OpenRCT2/OpenRCT2#14997]
**OS:** Windows 10 Professional x64
**Version:** 0.3.3
**Commit/Build:** 707d415 on development

So, I noticed something when I was setting up OpenRCT2 tonight (first time using), and noticed that the fountains in the demo videos don't animate; they're just static images.  However, in the original RCT2, they animate as they would in game, which incidentally, they do in in-game in OpenRCT2, as well.  I know this is a tiny thing, and by no means a game breaker, I felt I ought to bring it to your attention.

- [ No ] Reproducible in RCT2 (vanilla)?
- [ No ] Specific to multiplayer?

**Steps to reproduce:**
1. Sit through the demo videos (If you're able to jump to individual demo videos, it's the 8th one.)
2. The fountains are static in the scene, when they should be animating.

**Dump file**

**Screenshots / Video:**

**Save game:**

**Comment by duncanspumpkin** [comment]
Do you mean the title screen? Pretty sure it's due to being at the max animated elements.

**Comment by darth-crunchus** [comment]
> 
> 
> Do you mean the title screen? Pretty sure it's due to being at the max animated elements.

Yes, but in the vanilla game, they are animated on the title screen

**Comment by Gymnasiast** [comment]
We made some fixes to map animations since then, so I'm ging to assume that this has been solved.
